src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class FigureSupplier extends ColorSupplier {
+    public static final int Max_Size = 10;
+    public Figure getRandomFigure() { 
+    ColorSupplier color = new ColorSupplier();
+    String randomColor = color.getRandomColor();
+    int randomed = new Random().nextInt(5);
```

Comment:
The `Random` instance should be reused instead of creating a new one each time. Consider declaring a `Random` instance as a class-level variable.